docker/build.sh

```diff
@@ -69,21 +69,26 @@ SHELL ["bash","-c"]
 #
 # Additional python libs
 #
-RUN pip install nxpd $CUPY
-RUN conda install -y -c conda-forge dask-labextension recommonmark numpydoc sphinx_rtd_theme pudb \
+RUN source activate rapids \ 
+    && pip install nxpd $CUPY
```

Comment:
Hi Yi,

If I am not wrong, `nxpd` is installed by one of the dependencies., so it is not needed to manually install it.

Reply:
nxpd need to be installed explicitly. Here is the error I got if I take it out
```python
**---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-3-b5dd1bdfc799> in <module>
----> 1 import nxpd
      2 from gquant.dataframe_flow import TaskGraph
      3 
      4 # list of nodes composing the task graph
      5 task_list = [

ModuleNotFoundError: No module named 'nxpd'**
```

Reply:
👍